tests/infra_test.go

```diff
@@ -172,7 +172,7 @@ var _ = Describe("Infrastructure", func() {
 			By("Finding the prometheus endpoint")
 			pod, err = kubecli.NewVirtHandlerClient(virtClient).Namespace(tests.KubeVirtInstallNamespace).ForNode(nodeName).Pod()
 			Expect(err).ToNot(HaveOccurred(), "Should find the virt-handler pod")
-			metricsURL = fmt.Sprintf("https://%s:%d/metrics", pod.Status.PodIP, 8443)
+			metricsURL = fmt.Sprintf("https://%s:%d/metrics", tests.FormatIpForCurl(pod.Status.PodIP), 8443)
```

Comment:
Can't we use formatIpForUri ?

Reply:
Do you mean changing the name of the method?

Reply:
Best would be `formatIPForURL` ;)